homeassistant/components/thethingsnetwork/sensor.py

```diff
@@ -124,32 +124,32 @@ def __init__(self, hass, app_id, device_id, access_key, values):
         try:
             session = async_get_clientsession(self._hass)
             with async_timeout.timeout(DEFAULT_TIMEOUT, loop=self._hass.loop):
-                req = await session.get(self._url, headers=self._headers)
+                response = await session.get(self._url, headers=self._headers)
 
         except (asyncio.TimeoutError, aiohttp.ClientError):
             _LOGGER.error("Error while accessing: %s", self._url)
-            return False
+            return
```

Comment:
We should return something explicitly since we return the response at the end. Eg `return None`. We're checking the return value during platform setup.

Reply:
> We're checking the return value during platform setup.

Guess that I missed that change. In the past only the component setup was checked but not the platforms.

Reply:
Right